lib/msal-browser/src/error/NativeAuthError.ts

```diff
@@ -31,7 +31,15 @@ export const NativeAuthErrorMessage = {
     userSwitch: {
         code: "user_switch",
         desc: "User attempted to switch accounts in the native broker, which is not allowed. All new accounts must sign-in through the standard web flow first, please try again."
-    }
+    },
+    tokensNotFoundInCache: {
+        code: "tokens_not_found_in_internal_memory_cache",
+        desc: "Tokens not cached in MSAL JS internal memory, please make the WAM request"
+    },
+    nativeInternalCacheNotDefined: {
```

Comment:
Shouldn't be needed if we always make sure to instantiate the in memory storage object

Reply:
Agreed to keep the name.